### ISSUE TITLE
Want to add a username.json file as my 1st commit to the fest 21.

### ISSUE THREAD

**Issue by Sagun-png** [keshavsingh3197/HacktoberFest-2021#80]

**Comment by Sagun-png** [comment]
Want to add a username.json file as my commit to the fest 21.

**Comment by gargi111** [comment]
I Want to add a username.json file as my commit to the Hactoberfest 21

**Comment by RichieJacob342** [comment]
Want to add a username.json file as my first commit to the fest 2021

**Comment by shreya-gb** [comment]
Hello, @keshavsingh4522 I have added my PR under this issue.

PR Link: https://github.com/keshavsingh4522/HacktoberFest-2021/pull/329

**Comment by SanjanaPurbia** [comment]
Can I work on this issue??

**Comment by beatrizsmerino** [comment]
![Captura de pantalla 2021-10-13 a las 3 21 01](https://user-images.githubusercontent.com/14045148/137050559-413cd323-9c1e-4952-901c-1ef47856d5b1.png)

Hi!! @Sagun-png This issue was not accepted by the Hacktoberfest 2021 although it has the tag `hacktoberfest-accepted`

**Comment by Sagun-png** [comment]
Hi @beatrizsmerino 
Yah I also faced same issue, after the next day it got label. I guess digital ocean excluded this account repositories.

**Comment by ashwani29** [comment]
@Sagun-png do you mean you got your PR counted by hacktoberfest?

**Comment by Sagun-png** [comment]
> @Sagun-png do you mean you got your PR counted by hacktoberfest?

@ashwani29 No. That's what I said, hactoberfest is excluding this account PR's
